src/diagnostics/gravitywave_diagnostics.jl

```diff
@@ -0,0 +1,90 @@
+# This file is included in diagnostics.jl
+
+# Gravity-wave related diagnostics
+
+###
+# Eastward Non-Orographic Gravity-Wave Tendency 
+###
+compute_utendnogw!(out, state, cache, time) = compute_utendnogw!(
+    out,
+    state,
+    cache,
+    time,
+    cache.atmos.non_orographic_gravity_wave,
+)
+compute_utendnogw!(_, _, _, _, non_orographic_gravity_wave) =
+    error_diagnostic_variable("utendnogw", non_orographic_gravity_wave)
+
+function compute_utendnogw!(
+    out,
+    state,
+    cache,
+    time,
+    ::NonOrographicGravityWave,
+)
+if pkgversion(ClimaDiagnostics) >= v"0.2.13"
+    return cache.non_orographic_gravity_wave.u_waveforcing
```

Comment:
```suggestion
    return cache.non_orographic_gravity_wave.uforcing
```
(I think you know, just commenting here in case you missed it.)

Reply:
Thanks, this has been fixed locally. :)